obfuscator.js

```diff
@@ -24,6 +24,7 @@ function obfuscateCandidate(candidate) {
     const cand = SDPUtils.parseCandidate(candidate);
     if (cand.type !== 'relay') {
```

Comment:
Without code duplication it could be:
```
[ 'relatedAddress', 'ip', 'address'].forEach(type => {
   if (cand[type]) cand[type] = obfuscateIP(cand[type]);
}
```

Reply:
as you have noticed, I <3 what you call duplication.